web/src/components/base/Snackbar.tsx

```diff
@@ -0,0 +1,88 @@
+import CloseIcon from '@mui/icons-material/Close'
+import Alert, { alertClasses } from '@mui/material/Alert'
+import AlertTitle from '@mui/material/AlertTitle'
+import Button from '@mui/material/Button'
+import IconButton from '@mui/material/IconButton'
+import MUISnackbar from '@mui/material/Snackbar'
+import { styled } from '@mui/material/styles'
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+
+import { SNACKBAR_AUTO_HIDE_DURATION } from 'shared'
+
+const StyledMuiSnackbar = styled(MUISnackbar)(({ theme }) => ({
+  [theme.breakpoints.down('md')]: {
+    marginBottom: 40,
+  },
+  [theme.breakpoints.up('md')]: {
+    marginBottom: 46,
+  },
+}))
+
+const StyledAlert = styled(Alert)`
+  display: flex;
+  align-items: center;
+
+  .${alertClasses.action} {
+    padding: 0 8px;
+  }
```

Comment:
🤔 Is this necessary?

Reply:
yes, it is to align the button with the alert text

Reply:
I checked and it seems correctly aligned to me. In any case, alignment should be fixed using `alignItems` and/or `justifyContent` styles. Using a padding for alignment breaks if the padding is changed on only one component.

Reply:
There was slight misalignment. I will have a look again :+1: